src/index.tsx

```diff
@@ -109,11 +116,16 @@ class WatchParty extends React.Component {
               );
             }}
           />
+          <Route path="/discord/auth" exact>
```

Comment:
Can we make this a modal on the app rather than its own route? or is a new path required for the discord auth?

Reply:
The user will redirected to the URL you choose when generating the OAuth2 URL. In our case the user will be redirected to watchparty.me/discord/auth . So I think we need this new path.